src/css/base.scss

```diff
@@ -1,77 +1,74 @@
-:root {
-  --accent-color: #f9bb47;
-  --background: #1a353f;
-  --shadow: #1a1b25;
-  --background-light: #015492;
-  --light-text: #bfc0c0;
-  --box-background: #ffffff;
-}
+@import 'variables';
+@import 'mixins';
 
 * {
   box-sizing: border-box;
   margin: 0;
   padding: 0;
 }
 
-body {
+%flex_center-shared {
   align-items: center;
-  background-color: var(--background);
   display: flex;
+}
+
+body {
+  @extend %flex_center-shared;
+  background-color: $background_color;
   flex-direction: column;
   font-family: 'Noto Sans SC', sans-serif;
 }
 
 nav {
-  align-items: center;
+  @extend %flex_center-shared;
   background-image: linear-gradient(white, #e2e2e2);
   color: #2b8b9e;
-  display: flex;
-  font-size: 18px;
-  height: 50px;
+  font-size: $fontSize_18px;
```

Comment:
I think the variable name here is a bit too specific...we would like our variable names to be a little more generic for Sass so that we can adjust our values without needing to rename their variables. e.g. if 18px is the main body text size, I would call this variable something like `defaultBodySize` so that in case I decide to update the design and make the size 20px, I don't have a variable named 18px

Reply:
Including `fontSize` in the variable name is also a bit too specific. (I would equate this to naming your JavaScript variables with their data type). We don't need `fontSize` in our variable name because we'll see that we're using the variable as a value for the `font-size` property which is a dead giveaway of what we're using it for :)

Reply:
@brittanystoroz I see what you are saying with the naming. I will get these changed in this branch,